index.js

```diff
@@ -0,0 +1,5 @@
+function getRandomNumber() {
+  console.log(Math.floor(Math.random() * 1000));
```

Comment:
are you sure it is returning a number between 1 and 1000? What happens if `Math.random()` returns 0?